vars/main-srsran.yml

```diff
@@ -0,0 +1,88 @@
+k8s:
+  rke2:
+    version: v1.23.4+rke2r1
+    config:
+      token: purdue-k8s-rke2
+      port: 9345
+      params_file:
+        master: "deps/k8s/roles/rke2/templates/master-config.yaml"
+        worker: "deps/k8s/roles/rke2/templates/worker-config.yaml"
+
+  kubectl:
+    version: v1.23.15
+
+  helm:
+    version: v3.10.3
+
+core:
+  standalone: true		# set to false to place under control of ROC
+  data_iface: ens18
+  values_file: "deps/5gc/roles/core/templates/sdcore-5g-values.yaml"
+  ran_subnet: "172.20.0.0/16"	# set to empty string to get subnet from 'data_iface'
+  helm:
+    local_charts: false		# set chart_ref to local path name if true
+    chart_ref: aether/sd-core
+    chart_version: 1.1.0
+  upf:
+    ip_prefix: "192.168.252.0/24"
+    iface: "access"
+    mode: af_packet		# Options: af_packet or dpdk
+    default_upf:
+      ip:
+        access: "192.168.252.3/24"
+        core:   "192.168.250.3/24"
+      ue_ip_pool: "172.250.0.0/16"
+  amf:
+    ip: "10.76.28.113"
+
+srsran:
+  docker:
+    container:
+      gnb_image: aetherproject/srsran-gnb:rel-0.0.1
+      ue_image: aetherproject/srsran-ue:rel-0.0.1
+    network:
+      data_iface: ens18
+      name: host
+      subnet: "172.20.0.0/16"
+      bridge:
+        name: rfsim5g-public
+  simulation: true
+  gnb:
+    conf_file: deps/srsran/roles/gNB/templates/gnb_zmq.conf
+    ip: "172.20.0.2"
```

Comment:
Why do you need to fix this?

Reply:
for your reference https://github.com/opennetworkinglab/aether-gnbsim/blob/master/roles/simulator/tasks/start.yml#L26

Reply:
I just used the template from oai blueprint. Also currently the srsgnb/srsue deployed using host network and am working on running it on separate network. Once its fixed then i can use the mechanism you mentioned to retrieve gnb IP. Also i do see that while creating the container we are using this parameter to mention the IP address used for gnb container (in OAI blueprint). So we need to think about it (let me play around this and update).